### PR TITLE
Remove the measures that are Supplementary unit from the Ex…

### PR DESCRIPTION
**Pull request by alexdesi**
### What?
Remove the measures that are Supplementary unit from the Exporting from the UK table.

### Jira link
[hott-1353](https://transformuk.atlassian.net/browse/HOTT-1353)

### CODE REVIEW

File: app/models/measure_collection.rb [hunk 1]
```diff
@@ -10,6 +10,10 @@ def initialize(measures)
     @measures = measures.clone
   end
 
+  def exclude_supplementary_unit
+    self.class.new(measures.reject { |m| m.measure_type.description == 'Supplementary unit' })
```
Comment (willfish) [inline app/models/measure_collection.rb]: We 100% shouldn't use a changing description for this

File: app/models/measure_collection.rb [hunk 2]
```diff
@@ -10,6 +10,10 @@ def initialize(measures)
     @measures = measures.clone
   end
 
+  def exclude_supplementary_unit
```
Comment (willfish) [inline app/models/measure_collection.rb]:
```suggestion
  def without_supplemenatry_unit
    self.class.new(measures.reject(&:supplementary_unit?))
  end
```

**Review by willfish — CHANGES_REQUESTED** [review]
We 100% shouldn't ship this using a non-uniquely identifying description when we have the types available

File: app/models/measure_collection.rb (hunk 1 above)
Comment (alexdesi) [reply app/models/measure_collection.rb]: Yeh, I was not sure about it, but I could not find any info in the table `measure_types` and `measures` where the relevant info is stored (and not the description).

**Review by willfish — APPROVED** [review]

**Pull request merged**